hexa/datasets/queue.py

```diff
@@ -39,7 +39,7 @@ def download_file_as_dataframe(
     )
     download_url = generate_download_url(dataset_version_file)
     if mime_type == "text/csv":
-        return pd.read_csv(download_url)
+        return pd.read_csv(download_url, low_memory=False)
```

Comment:
What does that mean ? Maybe add a comment explaining why we set it as low_memory no ?